widgets/testimonial/styles/default.less

```diff
@@ -1,6 +1,5 @@
-@import "../../../base/less/mixins";
```

Comment:
This is where the preprocessing is happening - https://github.com/siteorigin/so-widgets-bundle/blob/develop/base/siteorigin-widget.class.php#L867-L875